app/src/main/java/pm/gnosis/heimdall/data/repositories/impls/WalletConnectBridgeRepository.kt

```diff
@@ -243,6 +225,61 @@ class WalletConnectBridgeRepository @Inject constructor(
             config.handshakeTopic
         }
 
+    private fun handleCustomCall(sessionId: String, session: Session, call: Session.MethodCall.Custom) {
+        sessionRequests[call.id] = sessionId
+        try {
+            when (call.method) {
+                "gs_multi_send" -> {
+                    handleMultiSend(sessionId, session, call)
+                }
+                else ->
+                    call.apply {
+                        rpcProxyApi.proxy(RpcProxyApi.ProxiedRequest(method, (params as? List<Any>) ?: emptyList(), id))
+                            .subscribeBy(onError = { t ->
+                                rejectRequest(id, 42, t.message ?: "Could not handle custom call")
```

Comment:
would it make sense to extract error codes? Maybe enum of some sort with comments. Those numbers alone are not very informative

Reply:
I think it is a random number (maybe 0 would be better), this can be improved, but is more walletconnect related than batched transactions